### PR TITLE
Qa

### PR DESCRIPTION
**Pull request by Amit-Sharma45**
this is qa

### CODE REVIEW

File: amitsharma.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+my name is amit sharma
```
Comment (Ashwinislk) [inline amitsharma.txt]: you can change  your project name

**Review by Ashwinislk — COMMENTED** [review]
review your file

**Review by Amit-Sharma45 — COMMENTED** [review]
file changed

**Review by Ashwinislk — COMMENTED** [review]
you can merge now

**Pull request merged**